src/python_testing/TC_OPSTATE_2_3.py

```diff
@@ -123,12 +124,13 @@ async def test_TC_OPSTATE_2_3(self):
         asserts.assert_equal(ret.commandResponseState.errorStateID, Clusters.OperationalState.Enums.ErrorStateEnum.kNoError,
                              "errorStateID(%s) should be NoError(0x00)" % ret.commandResponseState.errorStateID)
 
-        self.print_step(11, "Read OperationalState attribute")
-        operational_state = await self.read_mod_attribute_expect_success(endpoint=self.endpoint,
-                                                                         attribute=attributes.OperationalState)
-        logging.info("OperationalState: %s" % (operational_state))
-        asserts.assert_equal(operational_state, Clusters.OperationalState.Enums.OperationalStateEnum.kRunning,
-                             "OperationalState(%s) should be Running(0x01)" % operational_state)
+        if self.check_pics("OPSTATE.S.M.ST_RUNNING"):
```

Comment:
Similar to paused above, I think that this `if` should incorporate steps 10, 11 and 12.